assets/js/components/Square/index.js

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 
 const Square = (props) => (
-  <button className="square" onClick={props.onClick}>
+  <button className={props.status} onClick={props.onClick}>
```

Comment:
is status really apt for this? Classname makes more sense for what the name of thsi props val is